common/parser/parser_param.h

```diff
@@ -73,7 +73,7 @@ class ParserParam {
   // Returns the maximum allocated size of parser stacks or 0
   // if ResizeStacks() was never called.
   // This is useful to determine a reasonable default parser stack size.
-  long MaxUsedStackSize() const {  // NOLINT(runtime/int)
+  long MaxUsedStackSize() const {  // NOLINT
```

Comment:
We should really try to make this match the private member type, if possible.
I vaguely recall trying this and running into some compiler warnings when consuming this value, so call sites should be reviewed.

Reply:
Changed to size_t